json/json-quiz.md

```diff
@@ -580,3 +580,32 @@ printNullness(JSON.parse('{ "lemmings": "null" }'));
 - `->`
 - `::`
 - `.` <<<<---- Correct
+
+#### Q48. If a string contains line breaks, what should you do?
+
+- Wrap line breaks in single quotation marks
+- Make no changes, because whitespace is allowed
+- Replace any line breaks with `\r`
+- Replace any line breaks with `\n` <<<<---- Correct
+
+#### Q49. Which number types are available in javascript but not supported in json?
+
+- Fractional and Transcendental
+- Infinity or Rational 
+- Rational and Irrational
+- Infinity or NaN <<<<---- Correct
+
+#### Q50. How should a date value be stored in JSON?
+
+- As a string with quotes
+- As a string without quotes
+- As a string in ISO 8583 format 
+- As a string in ISO 8601 format <<<<---- Correct
+
+#### Q51. Is the JSON code shown here valid?
+![JSON object with key "Vehicles" containing an array of emojis](https://github.com/Ebazhanov/in-quiz-questions/blob/ad95968c724eefc5ee0559065dec17af644e1156/json/questionimages/Q51/Q51.jpeg)
```

Comment:
Should work with relative path (`questionimages/Q51/Qt1.jpeg`) instead of a link.
Then if the image changes it will be auto updated instead of a link (which I think might be commit specific)

Reply:
questionimages/Q51/Q51.jpeg is not working

Reply:
Looking at other repos and such, looks like everyone has .png as the image. Maybe jpeg linking is broken?
Even here: https://github.com/Ebazhanov/in-quiz-questions/commit/dbe9c6fd5458fc1e28356fc31cb183f07cb52518?short_path=e7fce5e#diff-e7fce5e9e3584c2c566460f121f9bf4cf6cbeca13945ec7a5cb4c2640ae12e20

Want to try with replacing the .jpeg with .png?

Reply:
For .png as well, it is not working. I refered this : [https://github.com/Ebazhanov/in-quiz-questions/edit/master/adobe-photoshop/adobe-photoshop-quiz.md] and copied full url for the image, extension of image has no impact.